public/app/features/datasources/state/reducers.ts

```diff
@@ -26,6 +33,13 @@ export const dataSourcePluginsLoad = createAction('dataSources/dataSourcePlugins
 export const dataSourcePluginsLoaded = createAction<DataSourceTypesLoadedPayload>(
   'dataSources/dataSourcePluginsLoaded'
 );
+export const dataSourceHistoryLoad = createAction('dataSources/dataSourceHistoryLoad');
```

Comment:
Why is this not 
```suggestion
export const dataSourceHistoryLoad = createAction<DataSourceHistoryLoadPayload>('dataSources/dataSourceHistoryLoad');
```

Reply:
Because if I had to define a `DataSourceHistoryLoadPayload` it would be empty. This is just an event to say that we started to load a data source history, see here the changes I make to the state when receiving the event: https://github.com/AgnesToulet/grafana/pull/77/files#diff-4a587b302935dd495fed051b131dedd039e490c34dcb16b09d8eaccaefc4afd0R89-R91

Reply:
Thx :)